scripts/fix_files.sh

```diff
@@ -81,8 +81,9 @@ if [[ $do_clang_format -eq 1 && -f ./.clang-format ]]; then
 	if [ $? -eq 0 ]; then
 		cf_version="$(clang-format --version)"
 		regex="clang-format version 7\.0\.0 \(tags\/RELEASE_700\/final[ 0-9]*\/WithWrappingBeforeLambdaBodyPatch\)"
-		if [[ ! "$cf_version" =~ $regex ]]; then
-			echo "Incorrect clang-format: Version 7.0.0 with WrappingBeforeLambdaBody patch required (found: $cf_version)"
+		regex2="clang-format version 11\.[ 0-9]*"
```

Comment:
I'll take baby steps and do merges by small increments as I'm very time-limited. Especially for pure cosmetic changes.